### ISSUE TITLE
[Mod Enhancement] Adding support for TFC

### ISSUE THREAD

**Issue by Xelbayria** [Rabbitminers/Extended-Cogs#97]
Take a look at the [DynamicCogWheelRenderer.java#L81](https://github.com/Rabbitminers/Extended-Cogwheels/blob/f08b80db29963825161447ea457d87b16f315db5/common/src/main/java/com/rabbitminers/extendedgears/cogwheels/DynamicCogwheelRenderer.java#L81). If I understand the code correctly, that should be how the wheel render itself using the texture of a _log or _stem. The ID is `modID:stripped_<type>_log`

TerraFirmaCraft's ID for woods IIRC is `tfc:wood/oak_log` or `tfc:wood/log/oak` (I'll check and update)

Let me know if you want me to make PR

**Comment by Rabbitminers** [comment]
> Take a look at the [DynamicCogWheelRenderer.java#L81](https://github.com/Rabbitminers/Extended-Cogwheels/blob/f08b80db29963825161447ea457d87b16f315db5/common/src/main/java/com/rabbitminers/extendedgears/cogwheels/DynamicCogwheelRenderer.java#L81). If I understand the code correctly, that should be how the wheel render itself using the texture of a _log or _stem. The ID is modID:stripped_<type>_log

Yep, this is re-used from how create generates textures for water wheels. Seems like they've added compatibility for TFC [like so](https://github.com/Creators-of-Create/Create/blob/e40b844aa1ade68c9de3949dfd67345b15cdda0c/src/main/java/com/simibubi/create/content/kinetics/waterwheel/WaterWheelRenderer.java#L122). We could probably use a very similar implementation. I'd be happy to add this myself when I've got some time but you're welcome to open a PR if you'd prefer :)

**Comment by Xelbayria** [comment]
👍🏻 Let me know if you need me to make PR

**Comment by Rabbitminers** [comment]
Im not sure when I'll next be able to get round to working on this, might be about a little bit so if you've got the time and dont mind a PR would be great. I'll and get the changes released asap

**Comment by Xelbayria** [comment]
Alright, I should be able to create PR tomorrow.
